src/visualization/recipes_makie.jl

```diff
@@ -108,7 +108,9 @@ iplot(u, semi; kwargs...) = iplot(wrap_array_native(u, semi), mesh_equations_sol
 # This initializes a Makie recipe, which creates a new type definition which Makie uses to create
 # custom `trixiheatmap` plots. See also https://makie.juliaplots.org/stable/recipes.html
 @Makie.recipe(TrixiHeatmap, plot_data_series) do scene
-  Makie.Theme(;)
+  Makie.Theme(
+    colormap = :thermal,
```

Comment:
Why change the default?

Reply:
It looks more like the default `:thermal` for Plots.jl. Couldn't find an exact match :(

Reply:
Should we change our default to `:plasma` for Plots.jl? I guess this should also be available for Makie.jl, since it's one of the three "big" colormaps from PyPlot (viridis, plasma, magma)

Reply:
Here are some "standard" scientific colormaps
![image](https://user-images.githubusercontent.com/12693098/130496339-1d996e99-16a5-40af-a7bf-84988d8c99ad.png)
Inferno is basically the standard of Plots.jl.

Reply:
Hm - let me try `:inferno` again. I think I misread http://makie.juliaplots.org/v0.15.1/documentation/colors/index.html - :inferno is actually under matplotlib.

Reply:
Ah, it works! Let me add :inferno as default to 100% match Plots.jl for now. Later, it might be nice to add a `default_colormap` function to make this configurable.